core/src/main/java/org/apache/calcite/rel/rules/CoreRules.java

```diff
@@ -388,9 +388,17 @@ private CoreRules() {}
   public static final IntersectToSemiJoinRule INTERSECT_TO_SEMI_JOIN =
           IntersectToSemiJoinRule.Config.DEFAULT.toRule();
 
+  /** Rule that translates a {@link Union} to {@link Filter}. */
+  public static final SetOpToFilterRule UNION_TO_FILTER =
```

Comment:
A better name would have been `UNION_FILTER_TO_FILTER`, but because we already have a rule for `MINUS`, we should keep this name, which has the same structure.

Reply:
Although the name of this implementation class has been changed (to SetOpToFilterRule), the logic of MINUS_TO_FILTER has not been changed and the name is still MINUS_TO_FILTER. Can I continue to keep UNION and INTERSECT consistent with it (UNOIN_TO_FILTER and INTERSECT_TO_FILTER)?

Reply:
Sorry, please ignore this reply